.github/workflows/tests-ubuntu.yml

```diff
@@ -0,0 +1,70 @@
+name: Tests - Ubuntu
+on: [push, pull_request]
+
+jobs:
+  tests-ubuntu:
+    name: Tests - Ubuntu
```

Comment:
This is super-minor, but Github shows status as Tests - Ubuntu / Tests - Ubuntu

Reply:
Right, seems hard to change though. If the workflow name is omitted, the full name of the file is used, while if the job name is omitted, the job key is used:
![no_names](https://user-images.githubusercontent.com/1731933/103222323-eecb5f80-4902-11eb-97cf-3b95aae6b04a.png)

I tried setting the same name for the two test files, hoping they would get grouped, but it wasn't the case:
![Same name - not grouped](https://user-images.githubusercontent.com/1731933/103222573-3a7e0900-4903-11eb-902b-f2bc7c895043.png)

Reply:
Again, a nitpick, but what do you think about using shorter names? This is how the popup looks like, when you click on a check mark near a commit: 

<img width="537" alt="изображение" src="https://user-images.githubusercontent.com/107893/103338576-9fbb3100-4aa0-11eb-9a76-978f18385a9f.png">

Proposal: instead of naming a job "tests-ubuntu", name it "tox". Keep workflow names as-is.

Reply:
Indeed, names are still too long, thanks for insisting on this. I just renamed both the workflows and the jobs, looks much cleaner to me, what do you think?
![ci-popup](https://user-images.githubusercontent.com/1731933/103373245-26790980-4ab3-11eb-9ecb-8981e2f420e9.png)